go/interpreter/lfvm/interpreter.go

```diff
@@ -259,7 +259,7 @@ func steps(c *context, oneStepOnly bool) (status, error) {
 
 		// Consume static gas price for instruction before execution
 		if err := c.useGas(staticGasPrices.get(op)); err != nil {
-			return statusError, err
+			return statusError, errNotEnoughStaticGas
```

Comment:
`useGas` can only fail with an `errOutOfGas`, do we care if this is static or dynamic ?

Reply:
I am not sure. wdyt? 
The reason why I renamed it, is because there are many situations where out of gas can be produced, and I wanted to be specific about this check being the one failing.

Reply:
why do you want to be specific about this one case failing ?